final/index.html

```diff
@@ -40,16 +40,14 @@ <h1 class="header__title">Weather PWA</h1>
           </div>
         </div>
         <div class="description">
-          <div class="feels-like">
-            <span class="value"></span><span class="scale">°F</span>
-          </div>
-          <div class="precip"></div>
```

Comment:
Oh, right. That's intended, too. The new API doesn't have a comparable data point for `precip` either. So I'm using sunrise and sunset, instead.